internal/command/volumes/create.go

```diff
@@ -99,11 +102,11 @@ func runCreate(ctx context.Context) error {
 		return client.GetAppBasic(ctx, appName)
 	})
 
-	if confirm, err := confirmVolumeCreate(ctx, appName); err != nil {
-		return err
-	} else if !confirm {
-		return nil
-	}
+	// if confirm, err := confirmVolumeCreate(ctx, appName); err != nil {
+	// 	return err
+	// } else if !confirm {
+	// 	return nil
+	// }
```

Comment:
Does this happen somewhere else now, or are we not wanting to prompt for confirmation?